tests/Service/PromotionsService/AbstractPromotionServiceTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Tests\BBC\ProgrammesPagesService\Service\PromotionsService;
+
+use BBC\ProgrammesPagesService\Domain\Entity\Brand;
```

Comment:
Is this used? You could run [Optimize Imports](https://www.jetbrains.com/help/idea/optimizing-imports.html) (`⌥⌘O`) to remove any that you're not using instead of going through 1 by 1.